src/TwabController.sol

```diff
@@ -470,7 +470,7 @@ contract TwabController {
       _vault,
       _currentDelegate,
       _to,
-      userTwabs[_vault][_from].details.balance
+      uint96(userObservations[_vault][_from].details.balance)
```

Comment:
Why is the `balance` stored in a `uint112` in `AccountDetails` and not in a `uint96`?
Seems strange to store it in a `uint96` in `Observation` and not in `AccountDetails`.

Reply:
To have the `AccountDetails` fit 256 bits perfectly. They were mismatched in v4 (uint208 and uint224) and fit perfectly to the single slot so I retained that - I'm not sure if that's necessary though.